pkgs/development/tools/misc/dialog/default.nix

```diff
@@ -1,4 +1,5 @@
-{ stdenv, fetchurl, ncurses
+{ stdenv, fetchurl
+, ncurses
```

Comment:
As you're splitting the import into multiple lines, it is better to do it for all of them.

Reply:
stdenv and fetchurl rarely change, they are mainly used for downloading and some general utilities.

The other lines are the real dependencies, and they are more likely to be added or removed. This way, it makes the commits clearer.

Reply:
Yes, I see your point. That said, I personally prefer it to be consistent with all the imports. But it is a personal choice.

Reply:
I would also change it to one per line for everything or everywhere multiple inputs per line but this minor.